### PR TITLE
Initialize the Operator using Operator SDK

### PR DESCRIPTION
**Pull request by wizhaoredhat**

### CODE REVIEW

File: Dockerfile.rhel9 [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Build the manager binary
```
Comment (bn222) [inline Dockerfile.rhel9:1]: Change the name of this file to `Dockerfile.rhel`

**Comment by bn222** [comment]
I would like to have had the chance to review this before it merged. Can you undo the merge?

**Comment by wizhaoredhat** [comment]
> I would like to have had the chance to review this before it merged. Can you undo the merge?

I wanted to work on the prow image builder, and we were having issues with the bot not exactly behaving correctly. Can you review it post merge? I will address your comment in a new PR. All the commands I used are present in the commit messages.

File: Dockerfile.rhel9 (hunk 1 above)
Comment (wizhaoredhat) [reply Dockerfile.rhel9:1]: Will make this change.

**Pull request merged**